### DISCUSSION TITLE
Trouble getting door sensors to link

### DISCUSSION THREAD

**Discussion by jeremysharp** [Koenkk/zigbee2mqtt#7316] [Devices]
I've got a couple of these https://www.aliexpress.com/item/1005002348194512.html but I'm having difficulty linking them. During the linking process, they don't get added, and occasionally I get a notification saying that it has left the network, despite not showing up in the interface at all. Does anyone know of any issues with these?

Home Assistant OS + ZZH stick

EDIT: It's finally just popped into the interface, but saying it's unsupported. Any way around this, if that's the case?

**Comment by DubhAd** [comment]
If you look at the [supported devices page](https://www.zigbee2mqtt.io/information/supported_devices.html) it has a link at the top telling you how to [support new devices](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html).

**Reply by jeremysharp** [reply]
How did I miss that? I will try that, thank you!

**Comment by jeremysharp** [comment]
I'm following the 'how to add new devices' above but it states I need to use the model id that's given in the unsupported log message. But I have 'undefined':

`Zigbee2MQTT:warn  2021-05-04 13:46:33: Received message from unsupported device with Zigbee model 'undefined' and manufacturer name 'undefined'`
?
